### PR TITLE
feat: avatar ownership transfer, prep for aura pid migrations and remaining funds into avatars

### PR DESCRIPTION
**Pull request by petrovska-petro**
Tackles #1298 

Ownership transfer:

```
brownie run scripts/avatars/transfer_ownership 
```

Funds migrations to new pids:

```
brownie run scripts/avatars/migrate_pids

brownie run scripts/avatars/migrate_pids sim
```

### CODE REVIEW

File: scripts/avatars/migrate_pids.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        kek_id = staking_contract.lockedStakes(private_vault, 0)[0]
+        private_vault.withdrawLocked(kek_id)
+
+        vault.print_snapshot()
+
+    if aura_migration:
+        total_assets = list(aura_avatar.totalAssets())
+
+        pids = list(aura_avatar.getPids())
+        if PID_AURA_EXCEPTION != 0:
```
Comment (gosuto-inzasheru) [inline scripts/avatars/migrate_pids.py:68]: what is the purpose of this? checking a constant versus a hardcoded value?

Comment (petrovska-petro) [reply scripts/avatars/migrate_pids.py]: as touched on our meeting, modify lang and add onchain checks on 2945209862e13fab95358a879da13b7593e38d71

**Review by jalbrekt85 — APPROVED** [review]
new pids lgtm, withdraws from old positions and migrates to new aura/convex pids

**Review by gosuto-inzasheru — COMMENTED** [review]
- withdraw treasury's bpt and wcvx positions to treasury
- withdraw aura avatar's bpt positions to treasury
- deposit all bpt positions to aura avatar
- deposit cvx position to convex avatar

is there a reason you chose to not withdraw the wcvx position from the convex avatar before redepositing? wouldnt it be better to have a single locked position versus two?

**Comment by petrovska-petro** [comment]
> is there a reason you chose to not withdraw the wcvx position from the convex avatar before redepositing? wouldnt it be better to have a single locked position versus two?

@gosuto-inzasheru since the last arg is `True` means that will deposit additional funds in existing kekid. [ref](https://github.com/Badger-Finance/badger-avatars/blob/main/src/convex/ConvexAvatarMultiToken.sol#L325-L331)

**Pull request merged**